proto/buf/alpha/registry/v1alpha1/admin.proto

```diff
@@ -40,6 +40,13 @@ message UniquenessPolicy {
   bool enabled = 1;
 }
 
+// GracePeriodPolicy is the policy for the review flow grace period on the server.
+message GracePeriodPolicy {
+  // The number of seconds of the grace period.
+  // Set to zero to disable the grace period.
+  uint32 grace_period_seconds = 1;
```

Comment:
considered a `google.protobuf.Duration` here, but a) don't need to represent negative durations, and b) don't need nanosecond precision.

A `uint32` is ~4 billion seconds, which is >100 years, which is more than we're ever going to need to set this value to.